pkg/common/api/crypto/api.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/framework/spi/crypto"
+)
+
+// package Crypto contains the consolidated Crypto interface to be used by the framework.
+// it will be created via Options creation of pkg/framework/provider/crypto (implementation of
+// pkg/framework/spi/crypto)
+
+// Crypto interface provides all crypto operations needed in the Aries framework
+type Crypto interface {
+	crypto.Crypto
+}
+
+// Creator method to create a new Crypto service
+type Creator func() (Crypto, error)
```

Comment:
I don't understand the purpose of this method. Packages will obtain the crypto service from the context.

Reply:
this is to allow the creation of an api.Crypto instance using any implementation (SPI.Crypto implementation is one the possiblities)

Reply:
I don't understand what this means to packages using crypto.

Reply:
you're, this doesn't belong in common... I will move this to spi crypto package and make the framework create an instance of spi Crypto (instead of common Crypto) to be added in the context

Reply:
done